### PR TITLE
fix: remove app management methods without app ID

### LINKED ISSUE #42: close_app and launch_app should be removed
Appium XCUITest driver 5.0.0 has removed these methods, and they now return errors. The only workaround right now is to install the previous driver version, 4.35.0.
(Appium UIAutomator2 driver still supports them, but they will certainly be removed there as well in the future)

### PR DESCRIPTION
**Pull request by TDL-EdgarsEglitis**
Fixes #42 
Also changes the Android versions of these methods to use the execute method extension instead.

### CODE REVIEW

File: lib/core/device.rb [hunk 1]
```diff
@@ -250,34 +250,28 @@ def close_app(action = nil)
   # launches the app specified by the Android app package / iOS bundle ID
   # defaults to the app under test if Value is not provided
   # Accepts:
-  #   Value (optional)
+  #   Value
   def launch_app(action)
+    raise "Application identifier is required!" unless action.key?("Value")
```
Comment (alvarolaserna) [inline lib/core/device.rb:255]: Maybe we can say also that it is requiered to be writen specificly in the `Value` field?

File: lib/core/device.rb [hunk 2]
```diff
@@ -250,34 +250,28 @@ def close_app(action = nil)
...
     end
   end
 
   # closes the app specified by the Android app package / iOS bundle ID
   # defaults to the app under test if Value is not provided
   # Accepts:
-  #   Value (optional)
+  #   Value
   def terminate_app(action)
+    raise "Application identifier is required!" unless action.key?("Value")
```
Comment (alvarolaserna) [inline lib/core/device.rb:269]: same here?

File: lib/core/device.rb (hunk 1 above)
Comment (TDL-EdgarsEglitis) [reply lib/core/device.rb]: Done

**Review by alvarolaserna — APPROVED** [review]

**Comment by alvarolaserna** [comment]
looks good to me, Ill merge :)

**Pull request merged**